package.json

```diff
@@ -47,121 +48,70 @@
     "underscore": "1.7.0"
   },
   "devDependencies": {
+    "autoprefixer": "6.3.6",
     "babel": "6.5.2",
     "babel-cli": "6.7.5",
     "babel-core": "6.7.6",
-    "babel-jest": "11.0.2",
     "babel-loader": "6.2.4",
     "babel-plugin-transform-runtime": "6.7.5",
     "babel-preset-es2015": "6.6.0",
+    "babel-preset-jest": "11.0.2",
     "babel-preset-react": "6.5.0",
+    "babel-preset-react-hmre": "1.1.1",
     "babel-runtime": "6.6.1",
-    "browser-sync": "2.12.3",
-    "del": "2.2.0",
-    "envify": "3.4.0",
+    "css-loader": "0.23.1",
     "eslint": "2.0.0",
     "eslint-loader": "1.3.0",
     "eslint-plugin-react": "2.6.4",
+    "extract-text-webpack-plugin": "1.0.1",
+    "file-loader": "0.8.5",
     "gulp": "3.9.1",
-    "gulp-autoprefixer": "3.1.0",
-    "gulp-changed": "1.3.0",
     "gulp-connect": "3.2.2",
-    "gulp-csso": "2.0.0",
-    "gulp-if": "2.0.0",
-    "gulp-imagemin": "2.4.0",
-    "gulp-less": "3.0.5",
-    "gulp-replace": "0.5.4",
-    "gulp-sourcemaps": "1.6.0",
-    "gulp-uglify": "1.5.3",
-    "gulp-util": "3.0.7",
+    "html-loader": "0.4.3",
+    "html-webpack-plugin": "2.16.0",
+    "image-webpack-loader": "1.7.0",
     "jasmine-reporters": "2.1.1",
     "jest-cli": "11.0.2",
     "jest-webpack-alias": "2.2.0",
     "json-loader": "0.5.4",
+    "less": "2.6.1",
+    "less-loader": "2.2.3",
     "localStorage": "1.0.3",
     "marked": "0.3.5",
-    "mkdirp": "0.5.1",
     "node-libs-browser": "1.0.0",
-    "run-sequence": "1.1.5",
+    "react-hot-loader": "1.3.0",
+    "postcss-loader": "0.8.2",
+    "react-hot-loader": "1.3.0",
```

Comment:
this is here twice.
Also would've loved to see the added features on a different PR (post this massive one)